e2e-tests/pkg/pbm/pbm_ctl.go

```diff
@@ -271,6 +271,11 @@ func (c *Ctl) PITRestore(t time.Time) error {
 	return err
 }
 
+func (c *Ctl) PITRestoreClusterTime(t, i uint32) error {
+	_, err := c.RunCmd("pbm", "restore", "--time", fmt.Sprintf("%d,%d", t, i))
```

Comment:
Can we use a separate flag for cluster time? it simplifies parsing and makes it obvious in usage and implementation

Reply:
I'd leave it like this for now. I don't want to expose this functionality make any UI commitments on this stage